### ISSUE TITLE
today private variable should not be static

### ISSUE THREAD

**Issue by IlanFrumer** [ChiperSoft/Kalendae#165]
When the date changes (at midnight) it won't update. This is because it only gets initialized once.

Source: https://github.com/ChiperSoft/Kalendae/blob/master/src/moment.ext.js#L19
```
today = Kalendae.moment().startOf('day');
```

**Comment by adion** [comment]
We've patched this locally so that `today` is a function call, which IIRC is only called 5 times by Kalendae.

```js
function today() {
    return moment().startOf('day');
}
```
...
```js
return moment(date).startOf('day').yearDay() >= today().yearDay();
```
etc.

**Comment by ChiperSoft** [comment]
This has been fixed in v0.6
